demos/dashboard/demo/settings.py

```diff
@@ -170,22 +171,28 @@ def ADMINS(self):
     DATABASE_NAME = get_env("DATABASE_NAME", default=PROJECT_NAME)
     DATABASE_USER = get_env("DATABASE_USER", default=PROJECT_NAME)
     DATABASE_PASSWORD = get_env("DATABASE_PASSWORD", default=PROJECT_NAME)
-
-    @property
-    def DATABASES(self):
-        """
-        Build the databases object here to allow subclasses to override specific values
-        """
-        return {
-            "default": {
-                "ENGINE": "django.db.backends.postgresql_psycopg2",
-                "HOST": self.DATABASE_HOST,
-                "PORT": self.DATABASE_PORT,
-                "NAME": self.DATABASE_NAME,
-                "USER": self.DATABASE_USER,
-                "PASSWORD": self.DATABASE_PASSWORD,
-            }
+    
```

Comment:
It probably makes sense for the demo to use sqlite, just need to make sure we don't have anything running this in the wild

Reply:
I have commented out the extra db fields